CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Changelog
+.# Changelog
```

Comment:
Thanks for the PR!

This change looks unintentional.

CI is failing to compile the `unrar-sys` on `unknown-linux-musl`, does it require installing dependencies with `apt`?

Reply:
Ooops, that definitely was unintentional.

As for `unrar-sys` on musl-based systems, [it may be non-trivial](https://github.com/clux/muslrust/issues/114), but I'll look into it.

Reply:
I've tried to reproduce the build error, but failed to achieve that - linking statically against musl works fine for me.
(disclaimer: I do not use nor have at hand an Ubuntu machine)

I suspect it may be related to the specific setup of github actions environment.
I've pushed a workaround that links system `g++` to `musl-g++` to see whether any further errors will arise during tests.

Reply:
Thank you for running CI again, it helped me to reproduce the error.

Main issue here is that:
1) musl-tools does not provide C++ runtime linked against musl
2) libstdc++ included in github runner environment is linked against glibc

Easiest solution would be not to compile `-musl` builds directly on the runner system, but use cross environment (which includes full C++ toolchain linked against musl).